### DISCUSSION TITLE
How long do the two Thread network partitions with different channels merge automatically?

### DISCUSSION THREAD

**Discussion by lancersky** [openthread/openthread#8420] [Q&A] [answered]
**Describe the bug** A clear and concise description of what the bug is.
In our test environment, the two partitions will not be automatically merged for an hour.

**Comment by jwhui** [comment, marked as answer]
Thread devices should periodically announce their current channel about once every 12 minutes. You can find the configuration in [src/core/config/announce_sender.h](https://github.com/openthread/openthread/blob/main/src/core/config/announce_sender.h).

Note that Thread devices rely on the Active Operational Dataset's Active Timestamp to determine which Thread network configuration is the latest. I would check if the Active Operational Datasets for the different partitions have different Active Timestamp values.

**Reply by xiaojiang2017** [reply]
Do we need to manually increase the Active Timestamp?
For example, in a Thread network, after powering off all FTD, then disable and modify the channel of OTBR.
After enabling OTBR, then power on all FTD. 
At this time, the OTBR partition did not merge with the FTD partition automatically, because their Active Timestamp is all 0.

**Reply by jwhui** [reply]
Any change to the Active Operational Dataset must also be accompanied with an increase in the Active Timestamp. The Active Timestamp is how Thread devices determine which Active Operational Dataset is newer.
